report-ng/app/src/components/method-details/details.ts

```diff
@@ -84,7 +84,10 @@ export class Details {
     }
 
     private _copyStackTraceToClipboard(stackTrace: IStackTraceCause[]) {
-        const msg = stackTrace.flatMap(cause => cause.stackTraceElements).join("\n");
+        let msg = "";
```

Comment:
@sbke-mms 
The stacktrace can extended and also formated like this:
```java
const msg = stackTrace
            .flatMap(cause => cause.className + ": " + cause.message + "\n    " + cause.stackTraceElements.join("\n    "))
            .join("\n");
```

Reply:
@martingrossmann adjusted the message accordingly